test/module/irohad/ametsuchi/block_query_test.cpp

```diff
@@ -91,6 +91,15 @@ class BlockQueryTest : public AmetsuchiTest {
     }
   }
 
+  void TearDown() override {
+    transaction->abort();
+
+    postgres_connection->deactivate();
```

Comment:
`pqxx::connection_base::disconnect()` contains `m_Conn = m_policy.do_disconnect(m_Conn);`
`pqxx::connection_base::deactivate()` also contains `  m_Conn = m_policy.do_disconnect(m_Conn);`
So does `pqxx::connection_base::close()`, and it is called in destructor, so manual calls are not necessary.